include/bluetooth/att.h

```diff
@@ -63,8 +63,53 @@ int bt_eatt_connect(struct bt_conn *conn, uint8_t num_channels);
 size_t bt_eatt_count(struct bt_conn *conn);
 
 #endif /* CONFIG_BT_TESTING */
+
```

Comment:
Do we want to make `bt_eatt_disconnect()` public also?

Reply:
I don't see a use case for allowing the application to disconnect an EATT bearer.
It may happen automatically, for example if a transaction has not completed in 30 seconds on an EATT bearer.
It is valid for testing purposes, as we need to support that an EATT bearer may be disconnected at any time.